tests/php/unit/Controller/ViewControllerTest.php

```diff
@@ -77,7 +77,7 @@ protected function setUp(): void {
 			$this->appointmentContfigService,
 			$this->initialStateService,
 			$this->appManager,
-			$this->userId,
```

Comment:
exclude the linter for the tests directory and leave test files untouched

Reply:
damn ok